loadtest/store/memstore/random.go

```diff
@@ -12,9 +12,13 @@ import (
 )
 
 var (
-	ErrEmptyMap    = errors.New("memstore: cannot select from an empty map")
-	ErrEmptySlice  = errors.New("memstore: cannot select from an empty slice")
-	ErrLenMismatch = errors.New("memstore: cannot select from a map, not enough elements")
+	ErrEmptyMap       = errors.New("memstore: cannot select from an empty map")
+	ErrEmptySlice     = errors.New("memstore: cannot select from an empty slice")
+	ErrLenMismatch    = errors.New("memstore: cannot select from a map, not enough elements")
+	ErrTeamNotFound   = errors.New("memstore: team not found")
+	ErrUserNotSet     = errors.New("memstore: user is not set")
+	ErrNoTeamFound    = errors.New("memstore: no team found")
```

Comment:
ErrTeamNotFound and ErrNoTeamFound ? What's the difference ?

Reply:
There's a subtle difference but there might be a better way to express it. 

- `team not found` means we were explicitly looking for a team (we were passed the teamId) and weren't able to find it.
- `no team found` means we needed to pick one or more teams but we couldn't return any cause there was none to pick.

Reply:
Maybe something like `empty teams`/`teams not set` is more apt for the 2nd case ?

Reply:
Sounds good.